fetch/src/2_process.r

```diff
@@ -0,0 +1,36 @@
+process_data <- function() {
```

Comment:
Same goes for the naming convention for this file. Since the script and the function(s) defined within it are part of the 2_process phase, we'd like this R file to be saved under `2_process/src` instead of `fetch/src/2_process.R`. It seems a bit funny to split all of the scripts out into different subfolders like this for now, but it will make more sense as you get into the next steps of setting up your pipeline.